app/js/appController.js

```diff
@@ -18,13 +18,19 @@ define(function(require) {
     gameWindow: function(id) {
       var module = require('modules/gameWindow/module');
 
-      App.activeModule && App.activeModule.stop();
-      App.activeModule = module;
-      App.activeModule.start({id: id});
+      // App.activeModule && App.activeModule.stop();
```

Comment:
I did it because browser history navigation didn't work, namely *back*/*forward* buttons: each time you push *back*/*forward*, browser changes the URL, and corresponding marionette route is fired.

As before, it would cause our **gameWindow** module to stop and start again every time you navigate back and forth. In the end it simply didn't work that way, so I changed that like "if module is not here yet, start it".